### ISSUE TITLE
Staging Review finding: No announcement of page content changing

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#116266]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Management Tool 2
**Product:** Claim Status Tool
**Feature:** VA Letters and Documents

## Findings details
**VA.gov Experience Standard - issue:** [User can identify an element and its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

When you click Edit, screen readers announce "Save" since that is where focus is. Screen reader users are not aware of the content changing and the edit form being shown.

<details><summary>Link, screenshot or steps to recreate</summary>

https://staging.va.gov/records/download-va-letters/letters/letter-page > Mailing address > Edit

https://github.com/user-attachments/assets/793f9441-f0f3-4b6a-813f-9d117cacf34c

</details>

## Recommended action

Use an ARIA live region so that screen reader users know the page content has changed.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 4.1.3 Status Messages - Level AA](https://www.w3.org/TR/WCAG22/#status-messages)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by bacitracin** [comment]
@dysbo Usually `aria-live` regions are scoped quite small and used just for status update messages. I think the original intention of this ticket might have been to include an empty aria-live region that upon clicking “Edit” populated a message like “Edit mode activated” or something like that.

Including the entire mailing address component in an `aria-live` could lead to some weird behaviors, especially since there are interactive elements inside the component. I’d suggest checking back with Shira on if wrapping the mailing address component works for them /  moving focus to the checkbox is adequate to communicate a change in context.
